### ISSUE TITLE
Duplicate class error since 0.10.0 (Stately)

### ISSUE THREAD

**Issue by svenjacobs** [ReactiveCircus/cache4k#34]
In one of my projects Dependabot wants to update cache4k from `0.9.0` to `0.10.0` but now the build fails with

```
Execution failed for task ':app:checkDebugDuplicateClasses'.
> A failure occurred while executing com.android.build.gradle.internal.tasks.CheckDuplicatesRunnable
   > Duplicate class co.touchlab.stately.HelpersJVMKt found in modules stately-common-jvm-2.0.0-rc1 (co.touchlab:stately-common-jvm:2.0.0-rc1) and stately-strict-jvm-2.0.0-rc1 (co.touchlab:stately-strict-jvm:2.0.0-rc1)
```

Note: This is an Android/JVM project, no multiplatform project.

**Comment by svenjacobs** [comment]
Probably related to [this commit](https://github.com/ReactiveCircus/cache4k/commit/1c09e74afdae144445cab4a9ae8afa07cea4b30b).

**Comment by ychescale9** [comment]
do you happen to have stately brought in by other library? I don't think you do but just checking 😃

**Comment by Milchschlumpf** [comment]
Got the same error in my multiplatform project. Checked my dependency tree too, stately is from cache4k.

**Comment by ychescale9** [comment]
Fixed in #36

**Comment by Awatar666** [comment]
```c++
``
```
